source/configure/configuration-settings.rst

```diff
@@ -649,8 +649,8 @@ You can optionally install `these dependencies <https://github.com/sajari/doccon
 
 .. note::
   - Document content search is available in Mattermost Server from v5.35, with mobile support coming soon. 
-  - Searching document contents adds load to your server. 
-  - For large deployments, or teams that share many large, text-heavy documents, we recommended you review our `hardware requirements <https://docs.mattermost.com/install/requirements.html#hardware-requirements>`__, and test enabling this feature in a staging environment before enabling it in a production environment.
+  - Searching document contents add load to your server. 
```

Comment:
In this context, ``adds`` is correct because what's being searched is the contents of documents. The verb ``add`` needs to match ``searching`` rather than ``contents``.

Reply:
Great catch! Changes have been implemented.